### PR TITLE
[tradeoff-analytics] Add support for "generate_visualization"

### PR DESCRIPTION
**Pull request by dolevdotan**
Added support to the generate_visualization query param

### CODE REVIEW

File: services/tradeoff_analytics/v1.js [hunk 1]
```diff
@@ -55,10 +57,11 @@ TradeoffAnalytics.prototype.dilemmas = function(params, callback) {
     options: {
       method: 'POST',
       url: '/v1/dilemmas',
-      body: omit(params,['metadata_header']),
+      body: omit(params,['metadata_header','generate_visualization']),
       headers: {
         'x-watson-metadata' : params.metadata_header
       },
+      qs: params.generate_visualization === false ? {'generate_visualization': false} : undefined,
```
Comment (germanattanasio) [inline services/tradeoff_analytics/v1.js]: Can we change

```
qs: params.generate_visualization === false ? {'generate_visualization': false} : undefined,
```

to be:
```
qs: pick(params, ['generate_visualization'] ),
```

**Comment by germanattanasio** [comment]
@dolevdotan can you update your dev branch(dolevdotan:dev) so that this PR reflects only your changes?

File: services/tradeoff_analytics/v1.js (hunk 1 above)
Comment (dolevdotan) [inline services/tradeoff_analytics/v1.js]: the reason I did it like that is that I wanted to go with a more 'strict'
approach, and accept only booleans. Maybe I should have added some
assertion to check this.

2015-12-24 18:36 GMT+02:00 German Attanasio Ruiz <notifications@github.com>:

**Comment by dolevdotan** [comment]
I have no idea how to do this... I did as you requested, forked and worked
in 'dev' branch. That's all I know how to do.
If you can give me simple instructions (git commands), I guess I could do
it...

2015-12-24 18:42 GMT+02:00 German Attanasio Ruiz <notifications@github.com>:

**Comment by germanattanasio** [comment]
You can do it using the github ui. Creating a pull request from wdc dev to
your fork dev

**Comment by dolevdotan** [comment]
ok, done.

**Pull request merged**